### DISCUSSION TITLE
Support for users?

### DISCUSSION THREAD

**Discussion by ljharb** [community/OSPO#61] [Dashboard]
I'd love to eventually be able to view this same data for my user account as well.

Even better would be if I could get a single dashboard that works across *every* org I maintain, as well as my own user-owned repos, as _well_ as the user-owned repos that I have write access on.

**Comment by ahpook** [comment]
Fair point @ljharb - Given that we started off building this more directly targeting OSPOs, we aimed at answering an organization's questions with this dashboard ("What's in the org I'm in charge of?"), rather than an individual's questions ("What's going on with all of my orgs?"). But I can see how it'd be useful to have an aggregate view across all the stuff you're responsible for.

**Reply by ljharb** [reply]
In my past roles building and managing OSPOs, we have more than one org to manage, so either way i'd need a cross-org aggregate view :-)

The user account part helps with the broader open source use cases, for which virtually every enterprise use case also applies.
